pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -64,6 +66,28 @@ sub _check_for_kernel_version {
     elsif ( ( $environment eq 'virtuozzo' ) || ( $environment eq 'lxc' ) ) {
         $self->add_info_advice( 'text' => ['Kernel updates are not supported on this virtualization platform. Be sure to keep the host’s kernel up to date.'] );
     }
+    elsif ( (@kernel_update) && ( $kc_kernelversion ) ) {
+        if ( kcare_kernel_version("check") eq "New version available" ) {
+            $self->add_bad_advice(
+                'text' => [
+                    'Kernel patched with KernelCare, but out of date. running kernel: [_1], most recent kernel: [list_and,_2]',
+                    $kc_kernelversion,
+                    \@kernel_update,
+                ],
+                'suggestion' => ['This can be resolved either by running ’/usr/bin/kcarectl --update’ from the command line to begin an update of the KernelCare kernel version, or by running ’yum update’ from the command line and rebooting the system.'],
+            );
+        }
+        else {
+            $self->add_info_advice(
```

Comment:
I just double checked my inbox to verify - when I was testing this the other day, I received an email when it was at "info" level.

Give me a few moments, I'll see if I can reproduce the behavior.